add repo_sury (#108)
Linuxfabrik/lfops: roles/repo_sury/README.md

# Ansible Role linuxfabrik.lfops.repo_sury

This role deploys the [Sury repository](https://deb.sury.org/).

Runs on

> * Debian11 (and compatible)

Reviewer: ```suggestion
* Debian 11 (and compatible)
```

> 

Reviewer: did you test debian 10 too?

## Tags

| Tag         | What it does                |
| ---         | ------------                |
| `repo_sury` | Deploys the Sury Repository |

## Optional Role Variables

| Variable | Description | Default Value |
| -------- | ----------- | ------------- |

> | `repo_sury__enabled_repos` | The package repos which should be enabled. If variable is unset, none of the Sury package repos will be activated. | unset |

Reviewer: ```suggestion
| `repo_sury__enabled_repos` | A list of package repos which should be enabled. If the list is empty, none of the Sury package repos will be activated. | `[]` |
```

Reviewer: would it make sense to make this a mandatory variable? the role does basically nothing without this variable set, right?

Reviewer: also, default to an empty list and add it to `defaults/main.yml`, so that the loop doesn't fail due to the undefined variable

| ---         | ------------                |
| `repo_sury` | Deploys the Sury Repository |

> ## mandatory Role Variables

Reviewer: ```suggestion
## Mandatory Role Variables
```
